hugr-model/tests/fixtures/model-constraints.edn

```diff
@@ -0,0 +1,15 @@
+(hugr 0)
+
+(declare-func array.replicate
+  (forall ?t type)
+  (forall ?n nat)
+  (where (copy ?t))
+  (where (discard ?t))
+  [?t] [(@ array.Array ?t ?n)]
+  (ext))
+
+(declare-func tuple.copy
+  (forall ?t type)
+  (where (copy ?t))
+  (where (discard ?t))
+  [(@ foo.tuple ?t)] [(@ foo.tuple ?t) (@ foo.tuple ?t)] (ext))
```

Comment:
What is `foo.tuple` ??

Reply:
It's some arbitrary custom constructor to trigger a particular case in import/export for more test coverage. `tuple` is for "historical" reasons since the example was different at first... Can rename it into something more nondescript.